### PR TITLE
Allow to change the relay's credentials subcommand

### PR DESCRIPTION
**Pull request by wozniakpl**

### CODE REVIEW

**Comment by wozniakpl** [comment]
@Mokto I see you're referenced in many PRs so I'll ping you here. I'd be thankful if you could have a look.

**Comment by Mokto** [comment]
Can you please update the minor version of the chart & "document" the new value in values.yaml?

**Comment by wozniakpl** [comment]
> Can you please update the minor version of the chart & "document" the new value in values.yaml?

It should be done.

**Comment by wozniakpl** [comment]
Just fixed the commit msgs for new CI checks. @Mokto could you please have a look again?

File: sentry/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sentry
 description: A Helm chart for Kubernetes
 type: application
-version: 22.1.1
+version: 22.2.0
```
Comment (Mokto) [inline sentry/Chart.yaml]: Can you remove that change please?

Comment (wozniakpl) [reply sentry/Chart.yaml]: Done

**Review by Mokto — APPROVED** [review]

**Pull request merged**